modules/con_enabling-config_management_plugins_in_an_argo_cd_cr.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="enabling-config_management_plugins_in_an_argo_cd_cr_{context}"]
+= Enabling Config Management Plugins in an Argo CD CR
+
+Argo CD provides support for Helm, Jsonnet, and Kustomize as built-in config management tools. To use a different config management tool, or to enable features not provided by the built-in config management tools, you can use the Config Management Plugin (CMP).
+
+In the {gitops-title} Operator, the Config Management Plugin (CMP) is specified as a sidecar container for the Argo CD repo server container. For more information, see Config Management Plugins.
```

Comment:
This looks a little redundant after reading the following passage

Reply:
Are you referring to the entire passage being redundant and want me to remove it from the content @varshab1210?

Reply:
Hmm, the last `Config Management Plugins` in that sentence should be a link to the upstream docs: https://argo-cd.readthedocs.io/en/stable/operator-manual/config-management-plugins/

Reply:
Fixed.